src/lib/pgp-key.cpp

```diff
@@ -2618,14 +2525,14 @@ pgp_key_t::add_sub_binding(pgp_key_t &                       subsec,
     pgp_signature_t sig;
     sign_init(ctx.rng, sig, hash, ctx.time(), version());
     sig.set_type(PGP_SIG_SUBKEY);
-    if (binding.key_expiration) {
-        sig.set_key_expiration(binding.key_expiration);
+    if (binding.expiration) {
```

Comment:
May be confusing - "what expiration?". IMHO keep key_expiration will be better.

Reply:
Thanks, renamed.